### PR TITLE
Publish Kubernetes Tentacle chart to Artifactory

### PR DESCRIPTION
**Pull request by APErebus**
Adds a github action workflow to package the kubernetes tentacle chart, version it with prerelease info and publish to Artifactory

Shortcut story: [sc-66951]

### CODE REVIEW

File: .github/workflows/kubernetes-tentacle-publish-chart.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Get branch names
+        id: branch_names
+        uses: OctopusDeploy/util-actions/current-branch-name@current-branch-name.0.1.0
+
+      - id: version
+        run: |
+          echo "CHART_VERSION=${{ steps.read_chart_yaml.outputs.version }}-${{ steps.branch_names.outputs.branch_name }}-$(date +'%Y%m%d%H%M%S')" >> $GITHUB_ENV
+          
+      #- name: Login to Artifactory
+      #  run: helm registry login packages.octopushq.com -u '${{ secrets.ARTIFACTORY_USERNAME }}' -p '${{ secrets.ARTIFACTORY_PASSWORD }}'
```
Comment (cailyoung) [inline .github/workflows/kubernetes-tentacle-publish-chart.yaml:33]: Please remove the endpoint here and use a secret

**Review by cailyoung — CHANGES_REQUESTED** [review]

File: .github/workflows/kubernetes-tentacle-publish-chart.yaml (hunk 1 above)
Comment (APErebus) [reply .github/workflows/kubernetes-tentacle-publish-chart.yaml]: done.

File: .github/workflows/kubernetes-tentacle-publish-chart.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: Publish Chart
+
+on:
+  pull_request:
```
Comment (cailyoung) [inline .github/workflows/kubernetes-tentacle-publish-chart.yaml:4]: Did you want a path filter on this?

Comment (APErebus) [reply .github/workflows/kubernetes-tentacle-publish-chart.yaml:4]: good call!

Comment (APErebus) [reply .github/workflows/kubernetes-tentacle-publish-chart.yaml:4]: dine!

**Review by cailyoung — APPROVED** [review]
LGTM, as far as the pipeline is concerned

**Review by MissedTheMark — APPROVED** [review]
Fix up that version issue and it'll be good to go 👍

**Comment by APErebus** [comment]
> Fix up that version issue and it'll be good to go 👍

fixed in 42f42e8

**Pull request merged**